docs/content/components/blankslate.md

```diff
@@ -129,7 +129,7 @@ There are a few variations of how the Blankslate appears: `narrow` adds a maximu
 | `title` | `String` | `""` | Text that appears in a larger bold font. |
 | `title_tag` | `Symbol` | `:h3` | HTML tag to use for title. |
 | `icon` | `Symbol` | `""` | Octicon icon to use at top of component. |
-| `icon_size` | `Symbol` | `:medium` | One of `:small` (`16`) and `:medium` (`24`). |
+| `icon_size` | `Symbol` | `:medium` | One of `:medium` (`24`) and `:small` (`16`). |
```

Comment:
IMO it is nicer to have these from smallest to largest. What do you think?

Reply:
Agreed, but that would require a lot more customized sorting comparator than here right? https://github.com/primer/view_components/pull/624/files#diff-b3a4f33411bc8a22b787fc10f0f06694f1487183b49e9add40bf17929a4c2209R7-R9

Reply:
Yeah, I don't know how we'd even write that. What if we added a `sort` option `one_of` and had it be false for just these cases?